src/main/scala/Validator.scala

```diff
@@ -91,9 +110,22 @@ class Validator(var tableCsvFile: URI, sourceUri: String = "") {
     errors = errors.concat(e)
   }
 
+  private def fetchPrimaryKeyString(list: List[Any]): String = {
+    var primaryKeyString = ""
+    for (i <- list) {
+      i match {
+        case listOfAny: List[Any] =>
+          for (j <- listOfAny) primaryKeyString += (j.toString + ",")
+        case _ => primaryKeyString += (i.toString + ",")
+      }
+    }
+    primaryKeyString.dropRight(1)
```

Comment:
A better way to do this is to convert the `List[Any]` to `List[String]` and then use Scala's `mkString` function to put the appropriate number of `,` in place for you.

Reply:
`mkString` is a bit like `String.join` in other languages.

Reply:
👍